### ISSUE TITLE
WCAG - Listings page

### ISSUE THREAD

**Issue by Spookyrumble** [NoroffFEU/agency.noroff.dev#820]
Non unique ID's and h-elemets order. Search is also missing a form label.
![image](https://github.com/NoroffFEU/agency.noroff.dev/assets/114398900/c9cf71b2-31e8-4fd0-853b-394bc8d36cc6)
![image](https://github.com/NoroffFEU/agency.noroff.dev/assets/114398900/2ea41f94-817b-457d-9751-242d3e4f0ad8)
![image](https://github.com/NoroffFEU/agency.noroff.dev/assets/114398900/d5ca927c-34f7-4644-b21f-084b87d0b2c9)

**Comment by nyolarraklay** [comment]
PR # https://github.com/NoroffFEU/agency.noroff.dev/pull/827

**Comment by nyolarraklay** [comment]
new PR 
https://github.com/NoroffFEU/agency.noroff.dev/pull/883/
